doc/rustc.md

```diff
@@ -162,18 +164,53 @@ in the following ways:
 # Translating the Rust standard libraries
 
 `mir-json` requires slightly modified versions of the Rust standard libraries
-that are simpler to formally reason about. These modified standard libraries
-are located under the `libs/` subdirectory. After installing `mir-json`, you
-can run the `translate_libs.sh` script to compile the standard libraries using
-`mir-json`.
-
-By default, `translate_libs.sh` compiles the libraries for your host machine's
-architecture. If you want to cross-compile for a different target, you can
-optionally set the environment variable `TARGET` to a [target
-triple](https://doc.rust-lang.org/nightly/rustc/platform-support.html) when
-running `./translate_libs.sh`. This is experimental and we have only tested
-`wasm32-unknown-unknown` to work; you might get build errors for other targets.
+that are simpler to formally reason about. These modified standard libraries are
+located under the `libs/` subdirectory. After installing `mir-json`, you can run
+the `mir-json-translate-libs` program to compile the standard libraries using
+`mir-json`. It will create a directory called `rlibs_real` with a Rust sysroot
+structure and a symlink `rlibs` into the subdirectory of `rlibs_real` which
+contains the build artifacts. The path to `rlibs` can then be used as the value
+of `CRUX_RUST_LIBRARY_PATH` or `SAW_RUST_LIBRARY_PATH`.
+
+`mir-json-translate-libs` can take some optional command line arguments:
 
 ```
-$ TARGET=wasm32-unknown-unknown ./translate_libs.sh
+mir-json-translate-libs
+Build the custom Rust standard libraries for mir-json
+
+USAGE:
+    mir-json-translate-libs [OPTIONS] [libs]
+
+ARGS:
+    <libs>    Directory containing custom Rust standard libraries [default: ./libs]
+
+OPTIONS:
+        --generate             Print a shell script instead of actually running the build
+    -h, --help                 Print help information
+    -o, --out-dir <OUT_DIR>    Directory to place rlibs and rlibs_real in [default: next to libs]
+        --target <TARGET>      Rust target triple to configure the libraries for
```

Comment:
I presume that the default value of `--target` is your host machine's target triple? If so, it would be worth including a `[default: ...]` part of this flag description, similar to the one that `--out-dir` has.